Sources/SwiftInspectorVisitors/Tests/TypeConformanceVisitorSpec.swift

```diff
@@ -0,0 +1,116 @@
+// Created by Dan Federman on 1/27/21.
+//
+// Copyright © 2021 Dan Federman
+//
+// Distributed under the MIT License
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+import Nimble
+import Quick
+import Foundation
+import SwiftInspectorTestHelpers
+
+@testable import SwiftInspectorVisitors
+
+final class TypeConformanceVisitorSpec: QuickSpec {
+  private var sut = TypeConformanceVisitor()
+  
+  override func spec() {
+
+    describe("visit(_:)") {
+      context("when a type conforms to a protocol") {
+        context("with only one conformance") {
+          beforeEach {
+            let content = """
+            class SomeObject: Foo {}
+            """
+            self.sut = TypeConformanceVisitor()
```

Comment:
Let's move this up a level to a beforeEach under the `describe` method

Reply:
dd3547e5838886fcb61168abf5c7a65d03437b9f